src/pages/index.tsx

```diff
@@ -1,59 +1,67 @@
-        <div className={styles.buttons}>
-          <Link
-            className="button button--secondary button--lg"
-            to="/docs/intro">
-            Docs
-          </Link>
-        </div>
-      </div>
-    </header>
-  );
+    const { siteConfig } = useDocusaurusContext();
+    return (
+        <header className={clsx("hero hero--primary", styles.heroBanner)}>
+            <div className="container">
+                <CometLogoColorClaim />
+                <p className="hero__subtitle">{siteConfig.tagline}</p>
+                <div className={styles.buttons}>
+                    <Link className="button button--secondary button--lg" to="/docs/intro">
+                        Docs
+                    </Link>
+                </div>
+            </div>
+        </header>
+    );
 }
 
 export default function Home(): JSX.Element {
-  const {siteConfig} = useDocusaurusContext();
-  return (
-    <Layout
-      title={`Hello from ${siteConfig.title}`}
-      description="Description will go into a meta tag in <head />">
-      <HomepageHeader />
-      <main>
-        <HomepageFeatures />
-          <HomePageOverview />
-      </main>
-    </Layout>
-  );
+    const { siteConfig } = useDocusaurusContext();
+    return (
+        <Layout title={`Hello from ${siteConfig.title}`} description="Description will go into a meta tag in <head />">
+            <HomepageHeader />
+            <main>
+                <HomepageFeatures />
+                <HomePageOverview />
+            </main>
+        </Layout>
+    );
 }
 
-
 function HomePageOverview(): JSX.Element {
     return (
         <section className={styles.features}>
             <div className="container">
-                <div className="row" style={{display: "flex", alignItems: 'center'}}>
-                    <div style={{flex:1, marginRight: 40}}>
-                        <p>COMET DSP is a digital experience platform that or orchestrates content, internal and external data, own and third party services, authentication and many more aspects of modern web and native applications</p>
-                        <p>It manages navigational structures as well as digital assets, workflows and custom user rights deriving from third-party identity providers. It is optimized for search engines on the one hand and provides APIs for intranets on the other hand. It is a multi-channel, multi-language and multi instance system</p>
-                        <p>It is cloudnative, microservice based and very modular. It scales when necessary. It is headless to provider maximum flexibility for the frontend</p>
+                <div className="row" style={{ display: "flex", alignItems: "center" }}>
+                    <div style={{ flex: 1, marginRight: 40 }}>
+                        <p>
+                            COMET DSP is a digital experience platform that or orchestrates content, internal and external data, own and third party
```

Comment:
you didn't change that but it should be DXP, right? (in another MR)

Reply:
yes, this whole paragraph should be changed (in another MR)